### PR TITLE
Update PnP tests/namespaces, API names and doc comments.

### PR DESCRIPTION
**Pull request by drwill-ms**

### CODE REVIEW

File: iothub/device/src/TelemetryMessage.cs [hunk 1]
```diff
@@ -34,6 +29,12 @@ public TelemetryMessage(string componentName = default)
             }
         }
 
+        /// <summary>
+        /// Gets or sets the <see cref="TelemetryCollection"/> for this <see cref="TelemetryMessage"/>
+        /// </summary>
+        /// <value>A telemetry collection that will be set as the message payload.</value>
+        public TelemetryCollection Telemetry { get; set; } = new TelemetryCollection();
```
Comment (drwill-ms) [inline iothub/device/src/TelemetryMessage.cs:36]: properties after ctors

File: iothub/device/src/TelemetryMessage.cs [hunk 2]
```diff
@@ -16,13 +16,8 @@ namespace Microsoft.Azure.Devices.Client
     public sealed class TelemetryMessage : MessageBase
     {
         /// <summary>
-        /// Gets or sets the <see cref="TelemetryCollection"/> for this <see cref="TelemetryMessage"/>
-        /// </summary>
-        /// <value>A telemetry collection that will be set as the message payload.</value>
-        public TelemetryCollection Telemetry { get; set; } = new TelemetryCollection();
```
Comment (abhipsaMisra) [inline iothub/device/src/TelemetryMessage.cs:22]: nit: property below constructor

Comment (abhipsaMisra) [reply iothub/device/src/TelemetryMessage.cs:22]: sorry, ignore

**Review by abhipsaMisra — APPROVED** [review]

**Pull request merged**